### ISSUE TITLE
Suggestion about `docker` group to use docker client without sudo.

### ISSUE THREAD

**Issue by supermomonga** [k0kubun/itamae-plugin-recipe-docker#1]
By default, to connect to the docker daemon, we need to run docker client with sudo (or, as root user). [According to the  docker official document](https://docs.docker.com/engine/installation/ubuntulinux/#create-a-docker-group), there is the official way to use docker client without sudo, by adding the user to the `docker` group.

Document says that it's optional operation so is not necessarily for the all people, but I think it's handy for most people if this itamae recipe can treat this operation.

**Comment by supermomonga** [comment]
FYI, currently I wrote a following code to provisioning my server, but It's tested only with Ubuntu 14.04.

```ruby
include_recipe "docker::install"
execute "usermod -aG docker #{node['user']['name']}" do
  not_if %`groups #{node['user']['name']} | tr -s ' ' '\n' | grep docker`
end
```

**Comment by k0kubun** [comment]
Helpful information :+1:

**Comment by k0kubun** [comment]
Added the support in the same way as you suggested and released as v0.2.0.
Thank you for creating this issue!

**Comment by supermomonga** [comment]
Awesome! :tada:
It looks so 便利.
